components/SplitsDisplayTab.vue

```diff
@@ -14,11 +14,19 @@
       <loading-switch v-model="graphMedianAttemptHline" class="mb-2">
         Display median attempt's times as a horizontal line
       </loading-switch>
+      <b-col cols="10" offset="1">
+        <hr/>
+      </b-col>
+      <b-col class="text-center mt-3">
```

Comment:
![image](https://user-images.githubusercontent.com/71961459/173405311-1a61fdd8-2d25-41b4-be82-5bb93b859bb5.png)
if it's just a div it looks like this ^
![image](https://user-images.githubusercontent.com/71961459/173405436-4f4424d2-3951-44a4-9311-0feb9592640b.png)
with the b-col it's equal to the line above, so i thought it would look better